src/renderer/ytmview/scripts/hookplayerapievents.script.js

```diff
@@ -14,8 +14,9 @@
     const volume = state.player.volume;
     const adPlaying = state.player.adPlaying;
     const muted = state.player.muted;
+    const shuffled = !!document.querySelector("ytmusic-app-layout>ytmusic-player-bar").querySelector("ytmusic-player-bar[shuffle-on] .shuffle.ytmusic-player-bar");
```

Comment:
Can we swap this to a state store property like the rest of the information about the state. We avoid doing any DOM querying and rely on YTMs internal state store to provide us information.